.github/workflows/release.yml

```diff
@@ -0,0 +1,58 @@
+name: Release Workflow
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: "12.x"
```

Comment:
Why exactly is java needed for this actions?

Reply:
Flutter requires Java to build the Android app, even if it is built using Kotlin